_topic_maps/_topic_map.yml

```diff
@@ -105,7 +105,7 @@ Topics:
   Distros: openshift-enterprise
 - Name: Admission plugins
   File: admission-plug-ins
-  Distros: openshift-enterprise,openshift-aro
+  Distros: openshift-enterprise,openshift-aro, openshift-origin
```

Comment:
```suggestion
  Distros: openshift-enterprise,openshift-aro,openshift-origin
```
Doesn't look like there should be a space.

Reply:
Whoops! Thank you!